server/app/app.go

```diff
@@ -198,6 +199,7 @@ func (app *App) Start(opts ...appOption) error {
 	linterRepo := analyzer.NewRepository(db)
 	testRepo := test.NewRepository(db)
 	runRepo := test.NewRunRepository(db)
+	trRepo := testrunner.NewRepository(db)
```

Comment:
minor: just for consistency with other resource's repo I would call it `testRunnerRepo`